### PR TITLE
zookeeper client example #70. Deploy zookeeper and use the kazoo zookeeper c...

### PR DESCRIPTION
**Pull request by arpras**
...lient to track processes.

### CODE REVIEW

File: examples/server_client/test_suites/zookeeper_test.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from kazoo.client import KazooClient
```
Comment (jdehrlich) [inline examples/server_client/test_suites/zookeeper_test.py]: probably a good idea to update setup.py and the license notice file to list dependency on kazoo

File: examples/server_client/test_suites/zookeeper_test.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      {'pid_keyword': "zookeeper",
+       'executable': zookeeper_exec_location,
+       'extract': True,
+       'post_install_cmds':runtime.get_active_config('zookeeper_post_install_cmds'),
+       'stop_command':runtime.get_active_config('zookeeper_stop_command'),
+       'start_command': runtime.get_active_config('zookeeper_start_command')})
+  runtime.set_deployer("zookeeper", zookeper_deployer)
+
+  zookeper_deployer.install("zookeeper",
+      {"hostname": "localhost",
```
Comment (jdehrlich) [inline examples/server_client/test_suites/zookeeper_test.py]: make this a config?

File: examples/server_client/test_suites/zookeeper_test.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+import perf
+import zopkio.runtime as runtime
+import zopkio.test_utils as testutilities
+import zopkio.adhoc_deployer as adhoc_deployer
+
+
+zookeper_deployer = None
+
+def start_zookeeper():
```
Comment (jdehrlich) [inline examples/server_client/test_suites/zookeeper_test.py]: do you want to move this to a deployment.py?

File: examples/server_client/test_suites/zookeeper_test.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+  #At 10 validate that ephemeral node exist that is the process is still running
+  assert zk.exists("/my/zookeeper_test/node1"), "process node is not found at 10 s when it is still running"
+  
+  #At 60 validate that process has terminated by looking at the ephemeral node
+  time.sleep(50)
+  assert not zk.exists("/my/zookeeper_test/node1"), "process node  not found at 60 s when it should have terminated"
+
+  #Terminate Zookeeper
+  zookeper_deployer.undeploy("zookeeper")
```
Comment (jdehrlich) [inline examples/server_client/test_suites/zookeeper_test.py]: should this also be in a deployment.py

File: examples/server_client/zookeeper_configs/zookeeper_config.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "client_exec_path": "AdditionClient/out/artifacts/AdditionClient_jar/AdditionClient.jar",
```
Comment (jdehrlich) [inline examples/server_client/zookeeper_configs/zookeeper_config.json]: do these make sense to have in this config?

**Comment by arpras** [comment]
Most of this is because I just piggy backed on the existing deployer and just used this as a new example.. let me create a new deployer for just this test then I can get rid of all the other bagage.

**Pull request merged**